module/src/components/calendarView/calendarView.component.tsx

```diff
@@ -1,113 +1,41 @@
 import * as React from 'react';
 
-import { use as useForm } from '../../hooks/form';
-import { Arrays } from '../../utils/arrays';
-import { ClassNames } from '../../utils/classNames';
+import { Calendar } from '../..';
 import { Dates } from '../../utils/dates';
-import { Maths } from '../../utils/maths';
-import { Button } from '../button';
-import { Select } from '../select';
-import { ICalendarHighlight, ICalendarHighlightParsed, IDay } from './calendarView.types';
-import { getDayOfWeekHeadings, getDays, getMonths, getYears } from './calendarView.utils';
+import { CalendarDisplay } from '../calendarDisplay/calendarDisplay.component';
 
-interface ICalendarViewProps {
-  /** (Date|string|number) The currently selected date as a string or a `Date` object. Will default to today. */
-  selectedDate?: Dates.DateLike;
-  /** (Date|string|number) An optional minimum selectable date. If passed, all dates before this date will not be selectable. */
-  min?: Dates.DateLike;
-  /** (Date|string|number) An optional maximum selectable date. If passed, all dates after this date will not be selectable. */
-  max?: Dates.DateLike;
-  /**
-   * (Date|string|number) An optional date to display a range to.
-   * - Will use the `selectedDate` as the other end of the range.
-   * - This date will show as the beginning of the range if `selectedDate` is later, or the end of the range if `selectedDate` is sooner.
-   * - Useful if using two calendar inputs to create a "from - to" date range selector, simply pass the selected date from one as the `rangeTo` date on the other and vice versa.
-   */
-  rangeTo?: Dates.DateLike;
-  /** An optional array of dates to highlight with optional class names to apply to each highlighted date. */
-  highlights?: ICalendarHighlight[];
```

Comment:
I think this would still wanna retain props like this that are for displaying stuff on the calendar, as long as they just get spread right onto the CalendarDisplay

Reply:
It has retained these props, they've just been moved into a reusable interface that this one is now extending.